src/components/ViewContainer/getCRUDActions.js

```diff
@@ -36,7 +36,9 @@ export const getCRUDActions = ({
       return response || { ok: true };
     })
     .catch(error => {
-      onError('delete', record, error);
+      if (error.status !== 409) {
```

Comment:
I think it makes sense to move this check to a variable to add more context on this.

Reply:
It's an exceptional case for just one error types (associations validation fails) so I cannot see how to move it to a variable and where to place it aside of here.